Src/TestTypeFoundation/EmptyEnum.cs

```diff
@@ -2,6 +2,6 @@
 {
     public enum EmptyEnum
     {
-        //this must not contain any values
+        // this must not contain any values
```

Comment:
IMHO, it'd be nice if we could enforce that comments start with upper case and end with period:

```
// This must not contain any values.
```

But I'll leave this up to you guys to device :smiley:

Reply:
No such rule so far is present in the library. If it appears - we'll definitely enable it 😉